cardano-cli/src/Cardano/CLI/Legacy/Options.hs

```diff
@@ -1365,6 +1366,26 @@ pLegacyShelleyBasedEra envCli =
               , Opt.help $ "Specify the Babbage era (default)" <> deprecationText
               ]
         ]
-      , maybeToList $ pure <$> envCliAnyShelleyBasedEra envCli
+      , maybeToList $ pure <$> envCliAnyEon envCli
       , pure . pure $ EraInEon ShelleyBasedEraBabbage
       ]
+
+pEra :: EnvCli -> Parser (Exp.Some Exp.Era)
+pEra envCli =
+  asum $
+    mconcat
+      [
+        [ Opt.flag' (Exp.Some Exp.BabbageEra) $
+            mconcat
+              [ Opt.long "babbage-era"
+              , Opt.help "Specify the Babbage era (default)"
```

Comment:
This will need to be updated because mainnet is now in the Conway era.

Reply:
I changed my mind while working on some deprecations in the cli. We are going to keep `legacy` and `latest`.